server/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -263,7 +263,7 @@ private DivTag renderBlockList(
                   "flex-row",
                   "gap-2",
                   "py-2",
-                  "px-4",
+                  "px-" + (4 + (level * 2)),
```

Comment:
(blocking) It's hard to quickly understand what's going on. Pull the calculation up to it's own variable and assign the magic numbers to some constants.

Reply:
Done!